ocfweb/docs/docs/staff/startertasks.md

```diff
@@ -0,0 +1,89 @@
+[[!meta title="Starter Tasks"]]
+
+Want to dive into learning about technical infrastructure at the OCF, but not
+sure where to start? Here are some self-paced tasks you can do on your own. Feel
+free to ask for help in our [Slack channel](https://ocf.io/slack) or in person
+during [staff hours](https://ocf.io/staffhours)!
+
+Tasks marked with an asterisk (*) require staff privileges. If you want to work
```

Comment:
You need to escape the asterisks otherwise Markdown will view them as italicizing.

Reply:
Ah, thanks